eng/versioning/external_dependencies.txt

```diff
@@ -145,8 +145,6 @@ org.nanohttpd:nanohttpd;2.3.1
 org.revapi:revapi-java;0.20.0
 org.revapi:revapi-maven-plugin;0.11.2
 
-
-
```

Comment:
Nit: If there's nothing changed in this file please don't submit it with empty line removal.

Reply:
done